wger/exercises/models.py

```diff
@@ -439,7 +439,7 @@ def exercise_image_upload_dir(instance, filename):
     """
     Returns the upload target for exercise images
     """
-    return "exercise-images/{0}/{1}".format(instance.exercise_base.id, filename)
+    return "exercise-images/{0}/{1}".format(instance.uuid, filename)
```

Comment:
This only changes the subfolder :)

Instead of  e.g. ``exercise-images/5/the_original_filename.jpg``, it should be ``exercise-images/5/abcd....jpg``. With the current code it would result in ``exercise-images/abcd..../the_original_filename.jpg``

I was also thinking that it would be useful to preserve the file extension, you can easily extract it with ``pathlib.Path(filename).suffix``. Just add it as a third parameter of the format call (and in the string like ``...{0}/{1}.{2}``)

Reply:
Hey, ah right I get you, will change it around and submit a new PR.

Also not sure why some of the CI actions failed, any ideas?

Reply:
No problems!  You can just push a new commit to the same branch, the PR will update.

And don't worry about the CI, I haven't updated the main server yet and it's trying to download some images to build the docker file

Reply:
Grand, so pushed new commit there. I wrote it in the format "exercise-images/{0}/{1}{2}", I was getting "..jpg" instead of ".jpg" at the end if I took "exercise-images/{0}/{1}.{2}".

Ran the tests and it failed 1 test in lines 208-210 in base_testcase.py. Test is related to image names, so assuming it failed because of the new format and all is well. I took a look in base_testcase.py to see if I could rewrite the test for the new format but it would be a bit beyond my skills at the moment, still getting the hang of OOP!

Reply:
You are right, those compares the image files. I'll take a look later (that part of the tests are really hairy, usually they are not like that)